### DISCUSSION TITLE
Why is it set to 1080P output but actually 2K?

### DISCUSSION THREAD

**Discussion by InvokerMing** [obsproject/obs-studio#7752] [Development Questions] [answered]
In the video settings, two sizes are both set 1920 * 1080. But when streaming, the websites(bilibili) show that it's 2560 * 1440.

**Comment by sebastian-s-beckmann** [comment, marked as answer]
This discussion appears to be a support request, for which we don't use GitHub Discussions.
Please make sure to visit the support sections on our [Discord server](https://discord.gg/obsproject) or [Forums](https://obsproject.com/forum).

Thank you!
